src/Post/Post.css

```diff
@@ -65,12 +65,20 @@ section.post-header {
   color: rgb(38, 38, 38);
 }
 
-section.img-container {
+section.media-section {
   grid-area: img;
+  overflow: hidden;
+}
+.media-container {
   display: flex;
-  flex-direction: column;
+  width: max-content;
+  overflow-y: hidden;
+  overflow-x: auto;
+  transition: transform 0.25s ease-in-out;
+}
+.media-container::-webkit-scrollbar {
+  display: none;
 }
```

Comment:
You can hide the scroll by doing `overflow:hidden` instead of `overflow:auto`.
You don't need 

```css
.media-container::-webkit-scrollbar {
  display: none;
}
```

Reply:
TIL.
Thanks